hosted-ce/etc/osg/image-config.d/30-remote-site-setup.sh

```diff
@@ -46,12 +46,9 @@ function fetch_remote_os_info {
 
 # Allow the condor user to run the WN client updater as the local users
 function setup_sudo_users {
-    users=$(get_mapped_users)
-    [[ -n $users ]] || errexit "Did not find any HTCondor-CE SCITOKENS user mappings"
-
     CONDOR_SUDO_FILE=/etc/sudoers.d/10-condor-ssh
```

Comment:
```suggestion
    users=$1

    CONDOR_SUDO_FILE=/etc/sudoers.d/10-condor-ssh
```

Something like this would help readability to help indicate what the function is expecting as its first arg